### PR TITLE
Archer adding group parser

### PR DESCRIPTION
**Pull request by maiorrulz**

### CODE REVIEW

File: Integrations/integration-ArcherRSA.yml [hunk 1]
```diff
@@ -434,16 +434,32 @@ script:
...
+                        group.forEach(function (gr) {
+                            groupValue = groupValue ? groupValue + ',' + gr['-name'] : gr['-name']
+                        })
+                    }
+                }
+                var toPush = {
                     FieldId: field['-id'],
                     FieldName: mapping[field['-id']].Name,
                     FieldType: field['-type'],
-                    Value: field['-value'],
```
Comment (dorsha) [inline Integrations/integration-ArcherRSA.yml]: Keep the line and change it to:
`Value: field[groupValue || '-value']`

File: Integrations/integration-ArcherRSA.yml [hunk 2]
```diff
@@ -434,16 +434,32 @@ script:
...
                     LevelId: field['-id']
-                });
+                }
+                if (groupValue) {
```
Comment (dorsha) [inline Integrations/integration-ArcherRSA.yml]: remove these lines

File: Integrations/integration-ArcherRSA.yml (hunk 1 above)
Comment (maiorrulz) [reply Integrations/integration-ArcherRSA.yml]: Did you mean - ` Value: groupValue || field['-value'] ` ?

Comment (dorsha) [reply Integrations/integration-ArcherRSA.yml]: Right

**Pull request merged**